cmd/openshift-install/gather.go

```diff
@@ -137,7 +137,7 @@ func extractHostAddresses(config *types.InstallConfig, tfstate *terraform.State)
 			logrus.Error(err)
 		}
 	case azuretypes.Name:
-		port = 2200
+		port = 22
```

Comment:
https://github.com/openshift/installer/pull/1867/commits/6f39659aeb11ec64a057545693d0775e73e295a4#diff-be4790dcfa83ca14e7942b4c3b50c5f9R128 already sets the port to 22, so you can skip assignment here...

Reply:
Removed!